source/_posts/2023-03-01-php-core-roundup-10.md

```diff
@@ -0,0 +1,320 @@
+  ```php
+    enum E {
+      const string TEST = "Test1";
+    }
+ 
+    trait T {
+      const string TEST = E::TEST;
+    }
+    
+    interface I {
+      const string TEST = E::TEST;
+    }
+
+    class C {
+      const string TEST = E::TEST;
+    }
+  ```
+  <br>
+
+*	**RFC Partially Accepted: [Readonly amendments](https://wiki.php.net/rfc/readonly_amendments) by Nicolas Grekas and Máté Kocsis💜**
+
+	This RFC attempts to address some of the shortcomings of PHP 8.1 readonly properties and 8.2 readonly classes.
+
+	This RFC proposed allowing `readonly` classes to be extended by non-readonly classes (currently not allowed, and causes a fatal error), and to allow reinitializing readonly properties during cloning (within the `__clone()` magic method).
+
+    During the two-part RFC vote, the first change of **allowing `readonly` classes to be extended by non-readonly classes was rejected**. The second change of **making it possible to reinitialize readonly properties during cloning was accepted**. 
+
+*	**RFC Implemented: [More Appropriate Date/Time Exceptions](https://wiki.php.net/rfc/datetime-exceptions) 💜**
+
+	RFC by Derick Rethans, proposed to introduce Date/Time extension-specific exceptions and errors. This detailed RFC suggests more specificity in the exceptions with exception classes such as `DateInvalidTimeZoneException`, and `DateMalformedPeriodStringException` as well as promoting some of the current PHP warnings to Error exceptions.
+
+    The [changes](https://github.com/php/php-src/pull/10366) are now merged.
+
+
+ 
+### Notable Mailing List Discussions
+
+- [Official Preprocessor](https://externals.io/message/119451)
+- [Deprecate ldap_connect with host and port as separate arguments](https://externals.io/message/119425)
+- [RFC proposal: function array_filter_list() to avoid subtle bugs/workarounds](https://externals.io/message/119444)
+- [RFC Proposal - Types for Inline Variables](https://externals.io/message/119470)
+- [Windows PECL build machine died](https://externals.io/message/117494)
+- [Class Re-implementation Mechanism](https://externals.io/message/119584)
+- [PHP code refactoring (was: include cleanup)](https://externals.io/message/119613)
+- [RFC Idea - json_validate() validate schema](https://externals.io/message/119631)
+
+
+## Merged PRs and Commits
+
+Following are some changes that did not go through an RFC process because they are either planned, bug fixes, or progressive enhancements. There are automated unit and integration tests for each of these changes, the PHP core developers review all pull requests.
+
+### Change Highlights
+ 
+ - Max Kellermann continued to clean-up and improve several extensions and the Zend engine.
```

Comment:
It looks like it's a controversial change, and not everyone agrees it actually improves things. Remove at all? Or remove "and improve".

Reply:
Yeah those recent emails were pretty controversial. I will leave to "clean-up" for now. Thank you.